tutorials/SequenceAlignment.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Structural alignment of proteins by sequence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the field of Bioinformatics is usual to perform a BLAST search in order to find\n",
+    "similar proteins to the one of your interest. If the structures of some of the results\n",
+    "are available, you might want to align them all together to see the differences. \n",
+    "These structures might not have the same number of residues, or the exact same residues,\n",
+    "or even the labeling of the atoms can be different, which makes a typical alignment function fail.\n",
+    "However, HTMD provides the function ```htmd.molecule.util.sequenceStructureAlignment```, which takes two proteins and\n",
+    "aligns both structures using their longest **sequence** aligment.\n",
+    "\n",
+    "In this example, we will use Dopamine receptor (PDB code: '3PBL') and Beta Adrenergic recptor (PDB code: '3NYA').\n",
+    "Both are GPCR proteins, so they share a great fraction of their sequence. We will use this feature to align their structures."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quick Example\n",
+    "\n",
+    "Adrenaline receptor will be used as the reference"
```

Comment:
Is Andrenergic the same as Adrenaline? Because in the rest of the tutorial you call it adrenaline receptor 
(bio-noob here)